App.tsx

```diff
@@ -11,6 +11,11 @@ import CaseContactCreateScreen from './src/screens/CaseContactCreateScreen';
 import CaseContactDetailScreen from './src/screens/CaseContactDetailScreen';
 import CaseContactListScreen from './src/screens/CaseContactListScreen';
 import LoginScreen from './src/screens/LoginScreen';
+import { NativeWindStyleSheet } from 'nativewind';
+
+NativeWindStyleSheet.setOutput({
```

Comment:
ooo what's this?

Reply:
Was NativeWind working for you without this code? I needed to add it in order for it to work.

Reply:
I didn't play with nativewind yet. lol.